telemetry/client.go

```diff
@@ -56,3 +57,44 @@ func (c *Client) PushReceivedMessages(filter transport.Filter, sshMessage *types
 		c.logger.Error("Error sending message to telemetry server", zap.Error(err))
 	}
 }
+
+func (c *Client) PushReceivedEnvelope(envelope *protocol.Envelope) {
+	c.logger.Debug("Pushing received envelope to telemetry server")
+	url := fmt.Sprintf("%s/received-envelope", c.serverURL)
+	postBody := map[string]interface{}{
+		"messageHash":    types.EncodeHex(envelope.Hash()),
+		"sentAt":         uint32(envelope.Message().Timestamp / int64(time.Second)),
```

Comment:
I think it would be good to also have `receivedAt` to compare how long it took for it to be received

Reply:
In case of messages, there is a createdAt field which is set by the service when the message is received. I did the same for envelopes:  https://github.com/status-im/telemetry/pull/12/files#diff-18714ff6aeb7898a2395af4afe27885d30c8c23e7855823d2cff7d9bb8b5d039R21

The timestamp is created when the service receives information about a new envelope.